frontend/src/scenes/onboarding/onboardingLogic.tsx

```diff
@@ -175,6 +185,28 @@ export const onboardingLogic = kea<onboardingLogicType>([
                 window.location.href = urls.default()
             } else {
                 actions.resetStepKey()
+                const includeFirstOnboardingProductOnUserProperties = values.user?.date_joined
+                    ? new Date(values.user?.date_joined) > new Date('2024-01-10T00:00:00Z')
+                    : false
+                eventUsageLogic.actions.reportOnboardingProductSelected(
+                    product.type,
+                    includeFirstOnboardingProductOnUserProperties
```

Comment:
I'm assuming this parameter means this is the first product the user is onboarding to. It's not clear that that condition is guaranteed. Checking for join date doesn't preclude the user having already onboarded to something else.

Reply:
How would they have onboarded to a different product before joining? 

In the event it uses $set_once, so no matter how many times it's set, it only keeps the first value it was ever sent.

Reply:
I see, then there’s not a problem.